spring-test/src/test/java/org/springframework/test/web/client/MockRestServiceServerTests.java

```diff
@@ -183,9 +183,11 @@ public void verifyWithTimeout() {
 		this.restTemplate.getForObject("/foo", Void.class);
 
 		assertThatThrownBy(() -> server1.verify(Duration.ofMillis(100))).hasMessage(
-				"Further request(s) expected leaving 1 unsatisfied expectation(s).\n" +
-						"1 request(s) executed:\n" +
-						"GET /foo, headers: [Accept:\"application/json, application/*+json\"]\n");
+				"""
```

Comment:
Please put the `"""` on the preceding line and reduce the indentation like you've done in `SimpleRequestExpectationManagerTests`.

Reply:
Fixed [95c2813](https://github.com/spring-projects/spring-framework/pull/29413/commits/95c28135c8804676c0951ff677a313d246fa409e)

Reply:
Test fails if we put `"""` on preceding line. [4bc8211](https://github.com/spring-projects/spring-framework/pull/29413/commits/4bc82118c71cf794df5578c522a4ed0261e49a6d)

I think the message have a newline at the end.

Reply:
Sorry, I was unclear. I meant the leading `"""`, not the trailing `"""`.

No worries. I'll take another look before merging.